uber/models.py

```diff
@@ -416,6 +417,25 @@ def setup_teardown(self):
 class Session(SessionManager):
     engine = sqlalchemy.create_engine(c.SQLALCHEMY_URL, pool_size=50, max_overflow=100)
 
```

Comment:
Docstrings need to go inside the method rather than before it.  If you do it this way, it won't be introspected as the docstring of the method.